### PR TITLE
add more entry to blog with former entry name

### PR DESCRIPTION
**Pull request by ebarboni**
This is to have more picture and news, all 2022 news back online. Note that style is no so great any idea welcome but another PR for that.

### CODE REVIEW

**Review by mbien — APPROVED** [review]
cool, even with pictures!

**Comment by mbien** [comment]
edit: can the welcome page render pictures? Are they in the rss feed?

**Comment by ebarboni** [comment]
yep it will, need merge to see if it's ok within the IDE

**Comment by ebarboni** [comment]
seems ok :p except the look and feel.
In the ide we have now the 5 last news so it's more dense.

File: netbeans.apache.org/src/content/blogs/announce-apache-netbeans-13-released.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+//     Unless required by applicable law or agreed to in writing,
+//     software distributed under the License is distributed on an
+//     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//     KIND, either express or implied.  See the License for the
+//     specific language governing permissions and limitations
+//     under the License.
+//
+
+= [ANNOUNCE] Apache NetBeans 13 Released
+Geertjan Wielenga
```
Comment (vieiro) [inline netbeans.apache.org/src/content/blogs/announce-apache-netbeans-13-released.adoc:21]: Hints:

```
:author: Geerjan Wielenga
:revdate: 2022-03-04
```

For more attributes see https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes-ref/#document-metadata-attributes

File: netbeans.apache.org/src/content/blogs/netbeans-pub-hangout-in-prague-pic1.png
Comment (vieiro) [inline netbeans.apache.org/src/content/blogs/netbeans-pub-hangout-in-prague-pic1.png:1]: D'oh! All those beers there! :+1:

**Review by vieiro — COMMENTED** [review]
Pretty cool. Even though this is merged I added some comments that can be helpful in the future.

**Comment by mbien** [comment]
looks good IMO:
image

the blog needs some style work though (more space between entries etc):
https://netbeans.apache.org/blogs/

**Pull request merged**